config/routes.rb

```diff
@@ -219,8 +219,8 @@
         resources :experience_points_records, only: [:index, :destroy]
         get 'invite' => 'user_invitations#new', on: :collection
         post 'invite' => 'user_invitations#create', on: :collection
-        get 'disburse_experience_points' => 'experience_points_disbursement#new', on: :collection
-        post 'disburse_experience_points' => 'experience_points_disbursement#create',
+        get 'disburse_experience_points' => 'experience_points/disbursement#new', on: :collection
```

Comment:
I'm not sure whether you can do a `collection do ...` over both routes. And since you also have a namespace for `ExperiencePoints`, is it possible to use `module: ...` also?

I'm out now so I'll need to read thru the docs if any suggestions were off (or if I remembered wrongly)